app/views/search/_filters.html.erb

```diff
@@ -0,0 +1,75 @@
+<% allow_delete ||= false %>
+<% allow_apply = true if allow_apply.nil? %>
+<% @active_filter ||= {} %>
+
+<div class="form-group">
+  <div class="form-group-horizontal">
+    <div class="form-group">
+      <%= label_tag :predefined_filter, 'Predefined Filters', class: "form-element" %>
+      <%= select_tag :predefined_filter, options_for_select([@active_filter[:name]],
+                                                            selected: @active_filter[:name]),
+          include_blank: true, class: "form-element js-filter-select", id: nil,
+          data: { placeholder: "" } %>
+    </div>
+    <button type="button" class="filter-clear button is-medium is-danger">Clear</button>
+    <% if allow_apply %>
+      <%= submit_tag 'Apply', class: 'button is-medium is-outlined', name: nil %>
+    <% end %>
+    <% if user_signed_in? %>
+      <button type="button" class="filter-save button is-medium is-filled">Save</button>
```

Comment:
Should we display the button but grayed out (disabled) and with a title set to indicate "this is not possible because you are not signed in"?

Reply:
Not sure - I personally prefer it the way it is now, but I'm open to changing it

Reply:
What is the current behavior?  (She says, hoping to avoid having to pull and set it up locally to find out. :-) )

Reply:
It's just Not There™️